packages/vue/src/components/FTooltip/FTooltip.vue

```diff
@@ -98,8 +98,8 @@ export default defineComponent({
          * - `h6`
          */
         headerTag: {
-            default: "div",
             required: false,
+            default: "",
             validator(value: string): boolean {
                 return ["div", "span", "h1", "h2", "h3", "h4", "h5", "h6"].includes(value);
```

Comment:
```suggestion
                return ["h1", "h2", "h3", "h4", "h5", "h6"].includes(value);
```

Samt uppdatera [`components.js`](https://github.com/Forsakringskassan/designsystem/blob/a2b3a23deaa46853270e179b1c82f69ceddee33b/packages/vue/htmlvalidate/elements/components.js#L566) med samma ändring.